pitch.md

```diff
@@ -7,14 +7,28 @@ _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+My language is for people who want to draw tableau trees, more specifically for students in CS 81.
+CS 81 requires trees that are rendered digitally and the recommended method for drawing the trees is to use Google Sheets.
+So students have to create text-boxes and arrows and drag them around to draw tree, which is quite cumbersome.
+A common alternative is to use LaTeX to typeset a tree.
+However, not everyone knows how to use LaTeX because the syntax is a bit challenging (and requires a lot of boilerplate). 
+Even if someone uses LaTeX for their other classes, they may not be familiar with the LaTeX packages for typesetting trees.
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+Logicians already use a DSL, LaTeX, for this purpose.
+A language has a steeper learning curve compared to using Google Sheets, but once someone has learned a language, it should be much easier to use the language than drawing a diagram.
+It seems like another DSL, with a simpler interface compared to LaTeX can improve a user's experience.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm taking CS 81 and I really dislike drawing trees on Google Sheets.
+I use LaTeX for my other classes, so my initial thought was to use it for CS 81 as well.
```

Comment:
I know how to use LaTeX, and I understand the basic structure of a tableau tree. However, I don't know any other programs that Renata can use. :(